frontend/src/components/Seats/index.tsx

```diff
@@ -1,13 +1,15 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { ThunkDispatch } from 'redux-thunk';
+import { IPostSeat } from '../../API/interfaces';
 import { IStore } from '../../store';
 import { createSeatAction, deleteSeatAction, updateSeatAction } from './actions';
 import Presentational from './Presentational';
 
 export interface ISeat {
-  id: string;
+  id: number;
   info: string;
+  name: string;
```

Comment:
This does not reflect seat in backend, seat_name is nullable and could thus be undefined